src/main/java/facade/UserFacade.java

```diff
@@ -11,13 +13,37 @@
 public class UserFacade {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(UserFacade.class);
+  private UserDAO userDao;
+
+  public UserFacade(){
+      userDao = new UserDAO();
+  }
+
+  private boolean validateUserCookie(String cookie){
+      //TODO: check cookie for valid user
+      String cookieUserName = null;
+      String cookiePassword = null;
```

Comment:
Why is there a `cookiePassword`? From what I understood I thought the cookie was a single string that was unique to each user.